apisix/plugins/prometheus.lua

```diff
@@ -23,6 +23,13 @@ local plugin_name = "prometheus"
 local default_export_uri = "/apisix/prometheus/metrics"
 local schema = {
     type = "object",
+    properties = {
+        prefer_type = {
```

Comment:
I mean using a boolean `prefer_name`. The type is confusing, why `id` & `name` are types?

Reply:
Ah boolean I didn't realize that. Yes that would be better.